### PR TITLE
REQUESTS-1022 Added verifier

### PR DESCRIPTION
**Pull request by Vasis3038**

### CODE REVIEW

File: verifier/__main__.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import requests
```
Comment (alexarlord-boop) [inline verifier/__main__.py]: # Invalid PULL Commits

** Invalid Commit Message: VERIFIER-1022 Added verifier **
! Message must start with code word in ['LEETCODE', 'GENERATOR', 'HEXNUMBER', 'TRIANGLE', 'ITERATOR', 'REQUESTS']

** Invalid Commit Message: Update __main__.py **
! Message must start with code word in ['LEETCODE', 'GENERATOR', 'HEXNUMBER', 'TRIANGLE', 'ITERATOR', 'REQUESTS']
! Message must contain group number in ['1021', '1022']
! Message must start with ['Added', 'Deleted', 'Fixed', 'Refactored', 'Moved']

** Invalid Commit Message: Update __main__.py **
! Message must start with code word in ['LEETCODE', 'GENERATOR', 'HEXNUMBER', 'TRIANGLE', 'ITERATOR', 'REQUESTS']
! Message must contain group number in ['1021', '1022']
! Message must start with ['Added', 'Deleted', 'Fixed', 'Refactored', 'Moved']

** Invalid Commit Message: Update __main__.py **
! Message must start with code word in ['LEETCODE', 'GENERATOR', 'HEXNUMBER', 'TRIANGLE', 'ITERATOR', 'REQUESTS']
! Message must contain group number in ['1021', '1022']
! Message must start with ['Added', 'Deleted', 'Fixed', 'Refactored', 'Moved']

** Invalid Commit Message: Delete links.txt **
! Message must start with code word in ['LEETCODE', 'GENERATOR', 'HEXNUMBER', 'TRIANGLE', 'ITERATOR', 'REQUESTS']
! Message must contain group number in ['1021', '1022']
! Message must start with ['Added', 'Deleted', 'Fixed', 'Refactored', 'Moved']

Comment (alexarlord-boop) [inline verifier/__main__.py:1]: # Invalid PULL Commits

** Invalid Commit Message: VERIFIER-1022 Added verifier **
! Message must start with code word in ['LEETCODE', 'GENERATOR', 'HEXNUMBER', 'TRIANGLE', 'ITERATOR', 'REQUESTS']

** Invalid Commit Message: Update __main__.py **
! Message must start with code word in ['LEETCODE', 'GENERATOR', 'HEXNUMBER', 'TRIANGLE', 'ITERATOR', 'REQUESTS']
! Message must contain group number in ['1021', '1022']
! Message must start with ['Added', 'Deleted', 'Fixed', 'Refactored', 'Moved']

** Invalid Commit Message: Update __main__.py **
! Message must start with code word in ['LEETCODE', 'GENERATOR', 'HEXNUMBER', 'TRIANGLE', 'ITERATOR', 'REQUESTS']
! Message must contain group number in ['1021', '1022']
! Message must start with ['Added', 'Deleted', 'Fixed', 'Refactored', 'Moved']

** Invalid Commit Message: Update __main__.py **
! Message must start with code word in ['LEETCODE', 'GENERATOR', 'HEXNUMBER', 'TRIANGLE', 'ITERATOR', 'REQUESTS']
! Message must contain group number in ['1021', '1022']
! Message must start with ['Added', 'Deleted', 'Fixed', 'Refactored', 'Moved']

** Invalid Commit Message: Delete links.txt **
! Message must start with code word in ['LEETCODE', 'GENERATOR', 'HEXNUMBER', 'TRIANGLE', 'ITERATOR', 'REQUESTS']
! Message must contain group number in ['1021', '1022']
! Message must start with ['Added', 'Deleted', 'Fixed', 'Refactored', 'Moved']

**Comment by Vasis3038** [comment]
asdas

File: verifier/__main__.py (hunk 1 above)
Comment (Vasis3038) [inline verifier/__main__.py:1]: 6969

Comment (Vasis3038) [inline verifier/__main__.py:1]: 6969

Comment (Vasis3038) [inline verifier/__main__.py:1]: 3228

Comment (Vasis3038) [inline verifier/__main__.py:1]: WRONG PREFIX in commit message/n/nALL WRONG in commit message/n/nALL WRONG in commit message/n/nALL WRONG in commit message/n/nALL WRONG in commit message

Comment (Vasis3038) [inline verifier/__main__.py:1]: WRONG PREFIX in commit message/nALL WRONG in commit message/nALL WRONG in commit message/nALL WRONG in commit message/nALL WRONG in commit message

Comment (Vasis3038) [inline verifier/__main__.py:1]: WRONG PREFIX in commit message

ALL WRONG in commit message

ALL WRONG in commit message

ALL WRONG in commit message

ALL WRONG in commit message

**Comment by Vasis3038** [comment]
sdfsd

**Comment by Vasis3038** [comment]
sdfsdfsdf

**Pull request merged**